sgx-tools/gen-qe-target-info.go

```diff
@@ -15,8 +17,12 @@ var generateQeTargetInfoCommand = cli.Command{
 EXAMPLE:
 For example, save the target information file about Quoting Enclave retrieved from aesmd:
 
-	# sgx-tools gen-qe-target-info --targetinfo foo`,
+        # sgx-tools gen-qe-target-info --targetinfo foo`,
 	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "quoteType",
+			Usage: "specify the SGX quote type such as epid for unlinkable, epid for linkable and ecdsa",
```

Comment:
> The sgx-tools has the responsibility to tell users which values are valid. Thus you need to document well the valid values rather than a short description.

Done, thanks. Plz see the sgx-tools/README.md in this pr.

Reply:
I mean this short help info should also document well about the exact value. I mean "epid for unlinkable" is not the value.

Reply:
Change to
```
specify the SGX quote type, such as epidUnlinkable, epidLinkable or ecdsa
```